src/library_async.js

```diff
@@ -134,7 +137,23 @@ mergeInto(LibraryManager.library, {
               Asyncify.exportCallStack.push(x);
               try {
 #endif
+#if ASYNCIFY == 1 && MEMORY64
+                // when rewinding, function is called without any arguments
```

Comment:
How about: 

```
// When re-winding, the arguments to a function are ignored.  For i32 arguments we
// can just call the function with no args at all since and the engine will produce zeros
// for all arguments.  However, for i64 arguments we get `undefined cannot be converted to
// BigInt`.
```

I think this means that we can wrap this entire block in `if (sig.includes('p') or sig.includes('j')`.   In fact we can pull that out of this lambda as:

`var saveArgs = sig.includes('p') or sig.includes('j');` up on line 121

Reply:
Exports don't have a `sig` field. Is there an easy way to add it? Here is how `test_fibers_asyncify` generates them:
```javascript
var ___wasm_call_ctors = () => (___wasm_call_ctors = Module['asm']['__wasm_call_ctors'])();
var _main = Module['_main'] = (a0, a1) => (_main = Module['_main'] = Module['asm']['__main_argc_argv'])(a0, a1);
var ___errno_location = () => (___errno_location = Module['asm']['__errno_location'])();
var _emscripten_stack_get_base = () => (_emscripten_stack_get_base = Module['asm']['emscripten_stack_get_base'])();
var _emscripten_stack_get_end = () => (_emscripten_stack_get_end = Module['asm']['emscripten_stack_get_end'])();
var _malloc = (a0) => (_malloc = Module['asm']['malloc'])(a0);
var _free = (a0) => (_free = Module['asm']['free'])(a0);
var _emscripten_stack_set_limits = (a0, a1) => (_emscripten_stack_set_limits = Module['asm']['emscripten_stack_set_limits'])(a0, a1);
```

Reply:
Ah yes, we only conditionally include the `.sig` attributes. 

See: create_em_js in emscripten.py and jsifier.js.   Looks like in both cases we want to also generate the sigs when `ASYNCIFY == 1 && WASM_BIGINT`

Reply:
That worked but did not make signatures for dynCall_ functions which are used in `test_fibers_asyncify` i can extract them from function name, if needed. As well as for invoke_ ones